auth/templates/base.html

```diff
@@ -1,9 +1,32 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <meta charset="utf-8">
         <title>Rebble Auth</title>
+        <meta name="description" content="Rebble auth's login page">
+        <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+        <link rel="alternate" type="application/rss+xml" title="Rebble" href="http://rebble.io/feed.xml">
+        <link rel="icon" type="image/x-icon" href="http://rebble.io/images/favicon.ico">
+        <link rel="stylesheet" href="http://rebble.io/css/common.css">
```

Comment:
Auth is served over HTTPS, so these won't work. I generally recommend using `//` instead of either `http://` or `https://` to just use whatever the page is already using. Please change all the `http://` references to just `//`.

More broadly, I'm not certain that pulling the CSS in like that is the best approach — it depends on how "common" that CSS really is.